speech-api/SpeechSynthesisUtterance-volume-attribute-test-manual.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>5.2.3 SpeechSynthesisUtterance volume attribute test - Manual</title>
+  <style>
+    div,
+    #test {
+      display: block;
+      width: 640px;
+      word-break: break-all;
+      padding: 4px;
+    }
+    #test,
+    #volume {
+      background: skyblue;
+      font-weight: bold;
+    }
+  </style>
+  <script>
+    const text = "hello universe";
+    const volumes = [0, 0.16666666666666666, 0.3333333333333333, 0.5, 0.6666666666666666, 0.8333333333333333, 0.9999999999999999];
```

Comment:
@foolip Is the suggestion what is referred to in these reviews as a "nit"? What is the fundamental difference between ` [0, 0.16666666666666666, 0.3333333333333333, 0.5, 0.6666666666666666, 0.8333333333333333, 0.9999999999999999]` and `[0, 0.2, 0.4, 0.6, 1]`? How something "looks" is subjective and not relevant to what something does or does not do. The purpose is to test the entire specified range. Could also test each fraction from 0 to 1 instead of only incrementing by ~1.3. Changed to the suggested range nonetheless.